GVFS/GVFS.Tests/Should/ValueShouldExtensions.cs

```diff
@@ -35,6 +35,12 @@ public static T ShouldEqual<T>(this T actualValue, T expectedValue, string messa
             return actualValue;
         }
 
+        public static T ShouldEqualWithin<T>(this T actualValue, T expectedValue, T delta, string message = "")

```

Comment:
nit: ShouldBeWithin?

Reply:
Well the intent of the function is to be the equivalent of the `ShouldEqual` function but for floating point numbers. Maybe it also has uses as a bounds checker but the idea of it is the mirror the `ShoulEqual` function for types where `==` and `.Equals()` are dangerous

Reply:
Why use a floating point number then?  Maybe it should be using `decimal`?

Reply:
I was unaware of `decimal`, that does seem to better fit the need better.